snap/snapcraft.yaml

```diff
@@ -67,9 +67,16 @@ plugs:
   # the service-specific directory under $SNAP_DATA/secrets.
   edgex-secretstore-token:
     interface: content
-    content: edgex-secretstore-token
     target: $SNAP_DATA/mount/secrets
 
+slots:
+  kuiper-config:
+    interface: content
+    source:
+      read: 
+        - $SNAP/etc/sources
+        - $SNAP/etc/connections
```

Comment:
Change to:
```suggestion
        - $SNAP_DATA/kuiper/etc/sources
        - $SNAP_DATA/kuiper/etc/connections
```
Then connect
```
snap connect edgex-ekuiper:kuiper-config edgexfoundry:kuiper-config
```
```
$ sudo snap run --shell edgex-ekuiper.kuiper
# sudo ls /var/snap/edgex-ekuiper/current/etc/sources -l
# ls /var/snap/edgex-ekuiper/current/etc/sources -l
total 12
-rw-r--r-- 1 root root 8137 Feb 11 14:42 edgex.json
-rw-r--r-- 1 root root  440 Feb 11 15:37 edgex.yaml <-- note the modification time
```
Disconnect and compare the time:
```
# ls /var/snap/edgex-ekuiper/current/etc/sources -l
total 12
-rw-r--r-- 1 root root 8137 Feb 11 14:49 edgex.json
-rw-r--r-- 1 root root 1221 Feb 11 14:49 edgex.yaml
```

Reply:
Thanks. How to validate the content of modified `edgex.yaml` is correct inside the snap?

Reply:
You mean other than manually checking with cat or vi?

Reply:
Please ignore my comment. I was in the wrong place.